shishito/library/modules/runtime/environment/browserstack_control_environment.py

```diff
@@ -17,17 +16,15 @@ def __init__(self, shishito_support):
 
         self.bs_api = BrowserStackAPI()
 
-        self.project_root = os.getcwd() # TODO:
-        self.timestamp = time.strftime("%Y-%m-%d_%H-%M-%S")
-        self.result_folder = os.path.join(self.project_root, 'results', self.timestamp)
-
     def call_browser(self, combination, capabilities=None):
         """ Starts browser """
 
         # get browser stack credentials
+        bs_credentials = self.shishito_support.gid('browserstack')
```

Comment:
explicit check for ":" is not good (it's already contained in split)
(see: Easier to ask for forgiveness than permission.)
```
try:
   bs_user, bs_password = self.shishito_support.gid('browserstack').split(':', 1)
except (AttributeError, ValueError):
   raise ValueError('Browserstack credentials were not specified! Unable to start browser.')
```